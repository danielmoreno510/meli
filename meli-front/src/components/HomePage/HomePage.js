import "./styles.scss";

const HomePage = () => <div className="home-page">Home Page</div>;

export default HomePage;
