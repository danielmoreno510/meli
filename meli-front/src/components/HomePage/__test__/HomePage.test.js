import React from "react";
import { shallow } from "enzyme";

import HomePage from "../HomePage";

describe("HomePage component", () => {
  describe("WHEN the component is render", () => {
    const wrapper = shallow(<HomePage />);

    it("THEN should display HomePage component", () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});
