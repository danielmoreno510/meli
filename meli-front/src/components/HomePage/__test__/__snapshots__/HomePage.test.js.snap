// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage component WHEN the component is render THEN should display HomePage component 1`] = `
<div
  className="home-page"
>
  Home Page
</div>
`;
